src/Ng/Jobs/LightningJob.cs

```diff
@@ -73,6 +73,13 @@ public override string GetUsage()
             sw.WriteLine($"      Compressed only: Azure Storage account name.");
             sw.WriteLine($"  -{Arguments.CompressedStorageContainer} <container>");
             sw.WriteLine($"      Compressed only: Container to generate registrations in.");
+            sw.WriteLine($"  -{Arguments.ContentIsFlatContainer} true|false");
+            sw.WriteLine($"      Boolean to indicate if the registration blobs will have the content from the packages or flat container.");
+            sw.WriteLine($"  -{Arguments.FlatContainerName} <the flat container name>");
+            sw.WriteLine($"      It is required when the ContentIsFlatContainer flag is true.");
```

Comment:
It is passed as argument in order to be able to change the flatContainer container name.